internal/connect/command_offset_describe.go

```diff
@@ -69,10 +69,13 @@ func (c *offsetCommand) describe(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("`--staleness-threshold` cannot be less than 5 seconds")
 	}
 
-	reFetchTimeout, err := cmd.Flags().GetInt32("re-fetch-timeout")
+	timeout, err := cmd.Flags().GetInt32("timeout")
 	if err != nil {
 		return err
 	}
+	if timeout <= 0 {
+		return fmt.Errorf("`--timeout` has to be a positive value")
+	}
```

Comment:
Please use flag type `Uint32()` for `--staleness-threshold` and `--timeout`

Reply:
Done